modules/client/src/connext.ts

```diff
@@ -343,42 +321,19 @@ export class ConnextClient implements IConnextClient {
   public resolveCondition = async (
```

Comment:
While we're refactoring I'd like to clean up some of these names. It's not obvious what `resolveCondition` does just from it's method name (what kind of condition?). Event the `transfer` method suggests an atomic operation where, in reality, there are two distinct steps. IMO two methods `createTransfer` and `resolveTransfer` are much more obvious. What do you think?

(If we do rename these methods, we should prob keep the old ones around too for backwards-compatibility but mark them as depreciated & flag for removal during our next breaking release)

Reply:
I held off on the interface breaking changes. I renamed the methods internally. We can do a separate cleanup around the interface later.